### DISCUSSION TITLE
Signin with passkey

### DISCUSSION THREAD

**Discussion by 2gn** [home-assistant/frontend#16982] [Accessibility]
It'd be comfortable if we could sign in to our instance(s) using [passkey](https://blog.google/technology/safety-security/the-beginning-of-the-end-of-the-password/).
It's a passwordless authentication method that is considered to be safer than passwords. iOS 17 implemented this as a builtin feature.

If you're interested, there are already some [web services](https://passkeys.directory) that support this.

**Comment by simplenotezy** [comment]
Hmm this page mentions passkey is supported: https://passkeys.directory/details/home-assistant/ - but can't seem to figure out how to enable it in HA

**Reply by 2gn** [reply]
I guess it's about [the community forum](https://community.home-assistant.io/) and not the instance itself, which I'm talking about.
